tests/adv_diff/bp_adv_diff_02_2d.mpirun=8.output

```diff
@@ -1,4 +1,4 @@
 Error in q (only in the fluid region) :
-  L1-norm:  0.09049706675
-  L2-norm:  0.02532477419
-  max-norm: 0.04455027832
+  L1-norm:  0.09483566835
+  L2-norm:  0.02611177469
+  max-norm: 0.04508145645
```

Comment:
I have run into this issue in SAMRAI before (it doesn't consistently calculate patch boundary locations - we should probably just patch SAMRAI ourselves). I have family visiting this weekend so I don't have time to really go through this until Monday, but it looks good at a glance except for this change - the errors here are still much higher than the serial case and this patch makes them higher still. Is this what we expect to happen?

Reply:
Good catch. I guess the output of the serial case was not updated.

Reply:
@drwells I get the following error norms:

Error in q (only in the fluid region) :
  L1-norm:  0.09483566835
  L2-norm:  0.02611177469
  max-norm: 0.04508145645

with single processor (serial) as well as multiple processors for tests/adv_diff/bp_adv_diff_02 test. I tested using MPICH - 3.3.2.

Reply:
> the errors here are still much higher than the serial case and this patch makes them higher still. Is this what we expect to happen?

The serial case in tests directory is a different problem: `bp_adv_diff_01_2d`. The test which was failing was `bp_adv_diff_02_2d`, whose output is only in parallel (with 8 processors) in the test directory.

Reply:
You are absolutely right! My mistake.